sys/include/net/nanocoap.h

```diff
@@ -499,6 +524,50 @@ size_t coap_put_option_block1(uint8_t *buf, uint16_t lastonum, unsigned blknum,
  */
 size_t coap_put_block1_ok(uint8_t *pkt_pos, coap_block1_t *block1, uint16_t lastonum);
 
+/**
+ * @brief   Encode the given string as option(s) into pkt
+ *
+ * Use separator to split string into multiple options.
+ *
+ * @post pkt.payload advanced to first byte after option(s)
+ * @post pkt.payload_len reduced by option(s) length
+ *
+ * @param[inout]  pkt         pkt referencing target buffer
```

Comment:
This should be `@param[in,out]` according to [doxygen](https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdparam)

Reply:
Sure thing.